tests/app/data/test_schema_validation.py

```diff
@@ -13,7 +13,20 @@
 
 class TestSchemaValidation(unittest.TestCase):
 
-    def test_schema(self):
+    def test_invalid_schema(self):
+        schema_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY, "schema/schema-v1.json"), encoding="utf8")
+        schema = load(schema_file)
+
+        file = "test_invalid_routing.json"
+
+        json_file = open(os.path.join(settings.EQ_SCHEMA_DIRECTORY + '/../../tests/app/data/', file), encoding="utf8")
```

Comment:
Other thing I thought about was having the schema in the same directory as the test and find it based on the test_schema_validation.py location